vpr/src/device/rr_graph_builder.cpp

```diff
@@ -54,3 +54,6 @@ void RRGraphBuilder::clear() {
 void RRGraphBuilder::set_node_coordinates(RRNodeId id, short x1, short y1, short x2, short y2) {
     node_storage_.set_node_coordinates(id, x1, y1, x2, y2);
 }
+void RRGraphBuilder::set_node_capacity(RRNodeId id, short new_capacity) {
```

Comment:
Let's use inline now. You can change the node_coordinate() to be inline as well

Reply:
I have inlined the both set_node_capacity and set_node_coordinates function.